lib/ejs.js

```diff
@@ -170,7 +171,11 @@ function includeFile(path, options) {
   if (!opts.filename) {
     throw new Error('`include` requires the \'filename\' option.');
   }
-  opts.filename = exports.resolveInclude(path, opts.filename);
+  if(path.substr(0,1) === "/" && typeof opts.root == "string" && opts.root != ""){
```

Comment:
Older runtimes don't allow treating strings as arrays. We don't have an explicit support policy -- maybe we need one.